frontend/src/pages/BYONImages/BYONImagesTable.tsx

```diff
@@ -401,31 +400,33 @@ export const BYONImagesTable: React.FC<BYONImagesTableProps> = ({ images, forceU
             <Tr>
               <Td colSpan={8}>
                 <Bullseye>
-                  <EmptyState variant={EmptyStateVariant.small}>
+                  <EmptyState variant={EmptyStateVariant.sm}>
                     <EmptyStateIcon icon={SearchIcon} />
                     <Title headingLevel="h2" size="lg">
                       No results found
                     </Title>
```

Comment:
`<EmptyStateHeader>` can be used here.

```suggestion
                   <EmptyStateHeader
                     titleText="No results found"
                     icon={<EmptyStateIcon icon={SearchIcon} />}
                     headingLevel="h2"
                   />
```